processing/src/main/java/io/druid/query/PrioritizedExecutorService.java

```diff
@@ -39,10 +41,11 @@
 import java.util.concurrent.ThreadPoolExecutor;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.TimeoutException;
+import java.util.concurrent.atomic.AtomicLong;
 
 public class PrioritizedExecutorService extends AbstractExecutorService implements ListeningExecutorService
 {
-  public static PrioritizedExecutorService create(Lifecycle lifecycle, ExecutorServiceConfig config)
+  public static PrioritizedExecutorService create(Lifecycle lifecycle, DruidProcessingConfig config)
```

Comment:
why not add the property to ExecutorServiceConfig? That keeps PrioritizedExecutorService more generic and not specific to druid processing.